docs/source/tutorials & examples/index.rst

```diff
@@ -0,0 +1,13 @@
+.. _tutorials & examples:
```

Comment:
Nit: could we rename this to `tutorials-and-examples/index.rst`? Just because working with the filename with spaces is a little tricky to parse/requires escaping the spaces on the command line etc